docs/book/user-guide/component-guide/model-deployers/mlflow.md

```diff
@@ -76,45 +76,40 @@ You can run predictions on the deployed model with something like:
 
 ```python
 from zenml.integrations.mlflow.services import MLFlowDeploymentService
-from zenml.steps import BaseParameters, Output, StepContext, step
+from zenml.steps import Output, StepContext
+from zenml import step
 from zenml.services.utils import load_last_service_from_step
 
 ...
 
 
-class MLFlowDeploymentLoaderStepParams(BaseParameters):
-    """MLflow deployment getter configuration.
-
-    Attributes:
+# Step to retrieve the service associated with the last pipeline run
+@step(enable_cache=False)
+def prediction_service_loader(
```

Comment:
Maybe take this from `examples/mlflow_deployment/steps/prediction_service_loader/prediction_service_loader_step.py` to be consistent with the example

Reply:
The mlflow example README also still has the old definition